Birdee/Parser.cpp

```diff
@@ -1129,25 +1129,27 @@ std::unique_ptr<FunctionAST> ParseFunction(ClassAST* cls)
 	current_func_proto = funcproto.get();
 
 	ASTBasicBlock body;
-
-	if (tokenizer.CurTok == tok_into)
-	{
-		//one-line function
-		tokenizer.GetNextToken();
-		unique_ptr<ExprAST> expr = ParseExpressionUnknown();
-		auto pos = expr->Pos;
-		unique_ptr<StatementAST> stmt;
-		if (!is_return_void)
-			stmt = make_unique<ReturnAST>(std::move(expr), pos);
+	if (!is_pure_virtual) {
```

Comment:
Can we raise some user-friendly compiling errors if user defines a pure_virtual function but also provides a body?